modules/sdk/project-templates/project-templates/build.gradle

```diff
@@ -160,6 +160,7 @@ test {
 	systemProperty "nodejs.npm.ci.registry", project.property("nodejs.npm.ci.registry")
 	systemProperty "nodejs.npm.ci.sass.binary.site", project.property("nodejs.npm.ci.sass.binary.site")
 	systemProperty "repository.url", System.properties["repository.url"]
+	systemProperty "rootProjectDir", rootProject.projectDir
```

Comment:
Wouldn't it be better to just pass a `repository.local.dir` system property instead?

Reply:
But how do I get that passed into system properties?

Reply:
I guess i can modify init.gradle?

Reply:
Simply copy how it's done in `build-buildscript.gradle`, and add the value as a system property in the `test` task.